subprojects/zlib.wrap

```diff
@@ -1,6 +1,6 @@
 [wrap-file]
 directory = zlib-1.2.11
-source_url = http://zlib.net/fossils/zlib-1.2.11.tar.gz
+source_url = https://downloads.sourceforge.net/project/libpng/zlib/1.2.11/zlib-1.2.11.tar.gz
```

Comment:
Mmm but this is libpng project??

What about `https://zlib.net/zlib-1.2.11.tar.gz` ? does it also time out for you?

Reply:
> Mmm but this is libpng project??

Same hash, and it's an official mirror (from Google Search cache).

> What about `https://zlib.net/zlib-1.2.11.tar.gz` ? does it also time out for you?

Hmm thankfully it doesn't 😄